tests/test_sp3.py

```diff
@@ -89,3 +187,29 @@ def test_velinterpolation(self, mock_file):
         r2 = sp3.getVelPoly(result, 2)
         self.assertIsNotNone(r)
         self.assertIsNotNone(r2)
+
+
+class TestMergeSP3(TestCase):
+    def setUp(self):
+        self.setUpPyfakefs()
+
+    def test_sp3merge(self):
+        # Create some fake files
+        file_paths = ["/fake/dir/file1.sp3", "/fake/dir/file2.sp3"]
+        self.fs.create_file(file_paths[0], contents=input_data)
+        self.fs.create_file(file_paths[1], contents=input_data2)
+
+        # Call the function to test
+        result = sp3.sp3merge(sp3paths=file_paths)
+
+        # Test that epochs, satellite, attrs data is as expected:
+        epoch_index = result.index.get_level_values("J2000")
+        sat_index = result.index.get_level_values("PRN")
+        # Verify
+        self.assertEqual(min(epoch_index), 229608000)
+        self.assertEqual(max(epoch_index), 774619500)
+        self.assertEqual(sat_index[0], "G01")
+        self.assertEqual(sat_index[-1], "R02")
+        self.assertEqual(result.attrs["HEADER"].HEAD.VERSION, "d")
+        self.assertEqual(result.attrs["HEADER"].HEAD.AC, "AIES")
+        self.assertEqual(result.attrs["HEADER"].HEAD.COORD_SYS, None)
```

Comment:
Might be worth adding another one ensuring that the merged file ended up with PV_FLAG=`P`
(GitHub doesn't seem to have a way of writing a suggestion that adds a line, without modifying an existing one)
```suggestion
        self.assertEqual(result.attrs["HEADER"].HEAD.COORD_SYS, None)
        self.assertEqual(result.attrs["HEADER"].HEAD.PV_FLAG, "P")
```